src/utils/dateUtils.ts

```diff
@@ -66,20 +66,21 @@ function validTime(calendarProps: any, date: Date) {
 
   return Object.keys(calendarProps).some(key => {
     if (/(Hours)/.test(key)) {
-      return calendarProps[key](getHours(date), date);
+      return calendarProps[key]?.(getHours(date), date) ?? true;
```

Comment:
The default is `true`, which will cause the time option to always be in a warning state.
![image](https://user-images.githubusercontent.com/1203827/227865137-08c12ca0-a0fc-4f7c-9dcc-7bcdd72c7f24.png)